test/built-ins/RegExp/escape/escaped-code-points.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2024 Leo Balter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-encodeforregexescape
+description: Encodes code points greater than 0xFF
```

Comment:
I think this is wrong. None of the conditions in step 5 apply to any of the listed characters, so they fall through to step 6, which just returns the characters without escaping them.

Reply:
agree, this test seems wrong to me.